ccd-definition/CaseEventToComplexTypes/JudgmentStatusDetails-JO-nonprod.json

```diff
@@ -0,0 +1,26 @@
+[
+  {
+    "ID": "JudgmentStatusDetailsList",
```

Comment:
I think we can rename all ID of types in this fiel to "JudgmentStatusDetails" like casefield ID. It contains one field of "judgmentStatusTypes" which itself is type of "JudgmentStatusDetailsList". So the current ID can be bit confusing.

Reply:
Renamed!